plugins/plugin-github/che-plugin-github-oauth2/src/main/java/org/eclipse/che/security/oauth/GitHubOAuthAuthenticator.java

```diff
@@ -30,23 +30,29 @@
 import java.net.URL;
 
 import static com.google.common.base.Strings.isNullOrEmpty;
-
 /** OAuth authentication for github account. */
 @Singleton
 public class GitHubOAuthAuthenticator extends OAuthAuthenticator {
     @Inject
-    public GitHubOAuthAuthenticator(@Nullable @Named("che.oauth.github.clientid") String clientId,
-                                    @Nullable @Named("che.oauth.github.clientsecret") String clientSecret,
-                                    @Nullable @Named("che.oauth.github.redirecturis") String[] redirectUris,
-                                    @Nullable @Named("che.oauth.github.authuri") String authUri,
-                                    @Nullable @Named("che.oauth.github.tokenuri") String tokenUri) throws IOException {
-        if (!isNullOrEmpty(clientId)
-            && !isNullOrEmpty(clientSecret)
-            && !isNullOrEmpty(authUri)
-            && !isNullOrEmpty(tokenUri)
+    public GitHubOAuthAuthenticator(@Nullable @Named("che.oauth.github.clientid") String clientId, //
```

Comment:
It is uncommon for existing codebase to use trailing slashes, can you elaborate on why they are needed?